data/virt_autotest/setup_dns_service.sh

```diff
@@ -1,18 +1,57 @@
 #!/bin/bash
-
-source /usr/share/qa/qa_test_virtualization/shared/standalone
-bridgeif="br123"
-bridgeip="192.168.123.1";
-dns_domain_forward="testvirt.net";
-dns_domain_reverse="123.168.192";
-dhcpd_lease_file="/var/lib/dhcp/db/dhcpd.leases"
 setup_log_file="/var/log/virt_dns_setup.log"
-
 if [ -e ${setup_log_file} ];then
 	rm -f ${setup_log_file}
 fi
 
-zypper_cmd="zypper --non-interactive in lsb-release bridge-utils bind bind-chrootenv bind-utils"
+if [ -z $1 ];then
+	echo "script usage: $(basename $0) [-f DNS forward domain name (testvirt.net)] [-r DNS reverse domain name (123.168.192)] [-s DNS server ip (192.168.123.1)] [-h help]" | tee -a ${setup_log_file}
+	exit 1
+fi
+
+while getopts 'f:r:s:h' OPTION; do
+  case "$OPTION" in
+    f)
+      dns_domain_forward="$OPTARG"
+      echo "The forward domain is $OPTARG" | tee -a ${setup_log_file}
+      ;;
+    r)
+      dns_domain_reverse="$OPTARG"
+      echo "The reverse domain is $OPTARG" | tee -a ${setup_log_file}
+      ;;
+    s)
+      bridgeip="$OPTARG"
+      echo "The DNS server is $OPTARG" | tee -a ${setup_log_file}
+      ;;
+    h)
+      echo "script usage: $(basename $0) [-f DNS forward domain name (testvirt.net)] [-r DNS reverse domain name (123.168.192)] [-s DNS server ip (192.168.123.1)] [-h help]" | tee -a ${setup_log_file}
+      exit 1
+      ;;
+    *)
+      echo "script usage: $(basename $0) [-f DNS forward domain name (testvirt.net)] [-r DNS reverse domain name (123.168.192)] [-s DNS server ip (192.168.123.1)] [-h help]" | tee -a ${setup_log_file}
```

Comment:
Please wrap this line into a help function and replace these dup lines with calling the help func.

One more suggestion: give default values for f:r:s if no customization needed when calling this dns script.